includes/Modules/Analytics/Measurement_Code_Injector.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Analytics\Measurement_Code_Injector
```

Comment:
Can we move this and all other new classes (except the main ~~`Shirshu_Class`~~ `Advanced_Tracking` class) into its own namespace `Google\Site_Kit\Modules\Analytics\Advanced_Tracking`? This way it's clearly separate from already existing code in the `Google\Site_Kit\Modules\Analytics` namespace.